helm/data-hub/values.yaml

```diff
@@ -92,6 +92,21 @@ dagAplicationConfigurationFiles:
    #   relativeMountedPath:  mount-dir/new-file-name2.extension
    #   key: mount-dir/new-file-name1.extension
 
+airflowSmtp:
```

Comment:
do we have to run an SMTP server? Is it for cloud portability?
https://console.aws.amazon.com/ses/home?region=us-east-1 is what we use elsewhere, talking to it through the SMTP protocol so that there is no local server to run.

Reply:
(coupled with https://mailtrap.io/ for testing)

Reply:
@giorgiosironi - can you help me create SMTP credentials, and put them in Vault for me under
names `username` and `smtp_password` under the `secret/containers/data-hub/smtp-credentials` namespace

Reply:
(sorted through Slack)